### PR TITLE
Print Shops Feature

### PR DESCRIPTION
**Pull request by zaaakk**
-Laying groundwork so that Evy can start inputting Print Shop information:
Added "shops" which is a editable static which acts as a splash page for the "shop list", a list of print shops. Currently print shops are based on the essay model. Still need to add test cases. 
-Lots of bug fixing for the essay view related works feature 
Still a few cases where there's a bug, need to investigate further.
-Fixed the weird bug on the Essay list where the notes weren't displaying properly

### CODE REVIEW

**Comment by bcail** [comment]
Is rome_app/templates/rome_templates/essays/shop_detail.html used? If not, please remove.

**Comment by bcail** [comment]
Please add some tests for the new shops views. In unit_tests/test_views.py, you could add
class TestPeopleViews(TransactionTestCase):
with a test for each new url/view. You can use the responses package to simulate the responses from the BDR (like other tests in test_views.py).

Feel free to ping me with any questions.

**Comment by zaaakk** [comment]
Hey Ben,
added some tests, let me know if they look ok. Shop_detail.html will be used, but currently is inaccessible as Elli wanted to give Evy a chance to write an introduction (for the new static page shops.html) and add some shops into database before it goes completely live. 
thanks again for the help -Zak

File: rome_app/urls_app.py [hunk 1]
```diff
@@ -28,6 +29,8 @@
     url(r'^essays/(?P<essay_slug>\w+)/$', views.essay_detail, name='specific_essay'),
     url(r'^people/$', views.biography_list, name='people'),
     url(r'^people/(?P<trp_id>\d+)/$', views.biography_detail, name='person_detail'),
+    url(r'^shop_list/$', views.shop_list, name='shops'),
```
Comment (bcail) [inline rome_app/urls_app.py]: This name of "shops" is already used above. Need to use a unique name. Be sure to update your test below with the new url name.

**Pull request merged**